commcare_connect/opportunity/views.py

```diff
@@ -967,11 +953,62 @@ def send_message_mobile_users(request, org_slug=None, pk=None):
     )
 
 
+# used for loading api key dropdown
+@org_member_required
+def get_api_keys(request, org_slug=None):
+    hq_server = request.GET.get("hq_server")
+    if not hq_server:
+        return HttpResponse(
```

Comment:
the strings here should be marked for translation (and elsewhere in the PR, just seeing it here first)